Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/Connection.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AmazonS3.DownloadObject
+{
+    /// <summary>
+    /// Connection parameters.
+    /// </summary>
+    public class Connection
+    {
+        /// <summary>
+        /// Authentication method to use when connecting to AWS S3 bucket. Options are pre-signed URL or AWS Access Key ID+AWS Secret Access Key.
+        /// </summary>
+        /// <example>AWSCredentials</example>
+        [DefaultValue(AuthenticationMethod.AWSCredentials)]
+        public AuthenticationMethod AuthenticationMethod { get; set; }
+
+        #region Pre-signed URL
```

Comment:
No need for region.

Reply:
It's easier (at least) for me to read the code when you can collapse pre-signed or AWS creds part out of sight in time to time. This doesn't affect the code.